src/view/placeholder.js

```diff
@@ -34,7 +34,14 @@ export function attachPlaceholder( view, element, placeholderText, checkFunction
 	}
 
 	// Store information about element with placeholder.
-	documentPlaceholders.get( document ).set( element, { placeholderText, checkFunction } );
+	documentPlaceholders.get( document ).set( element, {
+		placeholderText,
+		checkFunction,
+
+		// Store information if 'ck' class is already present and don't remove it when detaching placeholder.
+		// https://github.com/ckeditor/ckeditor5-image/issues/198#issuecomment-377542222
+		removeCkClass: !element.hasClass( 'ck' )
```

Comment:
Whether to remove the `ck` class when detaching the placeholder.

Reply:
"if 'ck' class is already present" is exactly what the code below is so it's unnecessary to additionally write this in a comment.